tools/gluten-it/common/src/main/scala/io/glutenproject/integration/tpc/Constants.scala

```diff
@@ -43,6 +43,8 @@ object Constants {
     .set("spark.sql.parquet.enableVectorizedReader", "true")
     .set("spark.plugins", "io.glutenproject.GlutenPlugin")
     .set("spark.shuffle.manager", "org.apache.spark.shuffle.gluten.celeborn.CelebornShuffleManager")
+    .set("spark.driver.extraClassPath", "/opt/hadoop/*")
+    .set("spark.executor.extraClassPath", "/opt/hadoop/*")
```

Comment:
After code rebase, these two configurations can be deleted, and then add ```.set("spark.celeborn.client.shuffle.compression.codec", "none")```.

Reply:
updated